linera-execution/tests/revm.rs

```diff
@@ -23,6 +23,11 @@ use linera_execution::{
 };
 use linera_views::{context::Context as _, views::View};
 
+fn get_operation(operation: impl alloy_sol_types::SolCall) -> Result<Vec<u8>, bcs::Error> {
```

Comment:
`serialize_operation`? `operation_to_bytes`?

In general, I'd use `get_` for accessing fields or maybe reading data from storage, but not for conversions or serialization.